schemas/+acq/checkFileversion.m

```diff
@@ -0,0 +1,18 @@
+
+function oldversion=checkFileVersion(fn)
+
+oldversion = true ;
+try
```

Comment:
this is failing for me when analyzing 

    M/Mouse/2016-03-15_13-56-05/AODAcq/2016-03-15_14-38-45_%d.h5

specifically the h5readatt fails so we fall through to the catch statement. i believe this experiment was recorded with the new format, though.